robusta_cli/main.py

```diff
@@ -284,6 +290,15 @@ def gen_config(
             "prometheusSpec": {"resources": {"requests": {"memory": "300Mi"}, "limits": {"memory": "300Mi"}}},
         }
 
+    if robusta_api_key:
```

Comment:
In the ticket there is requirement for:
```
In the "Local/Test" Install instruction sections (there are multiple sections in different pages) add the following to limit Holmes' resource request: "--set holmes.resources.requests.memory=512Mi" (Technical note: it will just be ignored in the user is not installing the UI)
```

Did we missed it?